storage-manager/src/S3Storage.cpp

```diff
@@ -225,9 +225,9 @@ S3Storage::S3Storage(bool skipRetry) : skipRetryableErrors(skipRetry)
   }
 
   ms3_library_init();
-  if (libs3_debug == "enabled")
+  if (verbose_enabled || libs3_debug == "enabled")
   {
-    ms3_debug(1);
```

Comment:
IMO that's wrong behaviour, you'll have ms3_debug turned off, if verbose is not enabled by command-line option but turned on in config file. 
Let's assume both option turn on verbose,
so it should be `ms3_debug(1);`